src/globals/scss/vendor/@carbon/layout/scss/_breakpoint.scss

```diff
@@ -1,65 +1,85 @@
+// https://github.com/twbs/bootstrap/blob/v4-dev/scss/mixins/_breakpoints.scss
 @import 'convert';
 
-$cell-padding: 1rem;
+$grid-cell-padding: rem(16px);
+$grid-gutter: rem(32px);
 
 // Initial map of our breakpoints and their values
-$breakpoints: (
+$carbon-grid-breakpoints: (
```

Comment:
This rename is for avoiding name conflict, wondering if such rename makes sense in `carbon-elements` side.

Reply:
This is partly why the name for breakpoints is `$grid-breakpoints` over in: https://github.com/IBM/carbon-elements/blob/master/packages/layout/scss/_breakpoint.scss#L8

Noticed this coming up when Security tried pulling it in 👍

Reply:
@joshblack Thank you for confirming the intent of recent `$breakpoints` to `$great-breakpoints` name change. Created: https://github.com/IBM/carbon-elements/pull/132